bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -53,6 +53,22 @@ type InitConfiguration struct {
 	// fails you may set the desired value here.
 	// +optional
 	LocalAPIEndpoint APIEndpoint `json:"localAPIEndpoint,omitempty"`
+
+	// CertificateKey sets the key with which certificates and keys are encrypted prior to being uploaded in
+	// a secret in the cluster during the uploadcerts init phase.
+	// +optional
+	CertificateKey string `json:"certificateKey,omitempty"`
```

Comment:
This field is not necessary given that CABPK does not rely on kubeadm for copying certificates into machines.